SPECS/sriov-network-device-plugin/sriov-network-device-plugin.spec

```diff
@@ -0,0 +1,37 @@
+Summary:        Plugin for discovering and advertising networking resources
+Name:           sriov-network-device-plugin
+Version:        3.4.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin
+Source0:        https://github.com/k8snetworkplumbingwg/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  golang
+
+%description
+sriov-network-device-plugin is Kubernetes device plugin for discovering and advertising networking
+resources in the form of SR-IOV virtual functions and PCI physical functions
+
+%prep
+%autosetup -p1
+
+%build
+go build -mod vendor -o ./build/sriovdp ./cmd/sriovdp/
+
+%install
+install -D -m0755 build/sriovdp %{buildroot}%{_bindir}/sriovdp
+install -D -m0755 images/entrypoint.sh %{buildroot}%{_bindir}/entrypoint.sh
```

Comment:
I'm a bit averse to the idea of putting something as generically named as `entrypoint.sh` in `/usr/bin`. Since this script is not meant to be run by an end user, and we don't need to have it in the `$PATH`, can we put this under `%{_datadir}/%{name}` instead?

Reply:
I followed the same convention as the multus spec (that package comes from the same open-source working group as this sriov package and that package has an upstream spec on OpenSUSE where the put the entrypoint.sh in the bindir - to give some backgorund, the script needed for the dockerfile build

Reply:
Can we rename the file to %{name}-entrypoint.sh, like we do in the multus spec?

Reply:
For multus, we have to rename `entrypoint.sh` again inside the container since otherwise it will break other upstream manifests consuming that as it expects the name to be the same. While I understand the motivation behind the rename, Is that the general new guidance?

Reply:
the script is now named accordingly with the following convention: %{name}-entrypoint.sh - all other points have been resolved and so this issue looks resolved from my end too. I would like to get this merged in asap as I can then move onto the next tasks with this plugin